doc/modules/model_evaluation.rst

```diff
@@ -1075,6 +1075,36 @@ Here is a small example of usage of this function::
     >>> label_ranking_average_precision_score(y_true, y_score) # doctest: +ELLIPSIS
     0.416...
 
+Ranking loss
+------------
+
+The :func:`label_ranking_loss` function computes the ranking loss which
+averages over the samples the number of pairs labels that are incorrectly
+ordered, i.e. true labels have a lower score than false labels, weighted by the
+the inverse number of false and true labels. The lowest achievable
```

Comment:
Is that the same as 1 - roc_auc apart from the weighting? If there is a relation to roc_auc it would be good to explain it.